.idea/externalDependencies.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
isn't `.idea` folder usually ignored?

Reply:
Not always. This is coming from the GitLive repo